cw/__init__.py

```diff
@@ -20,35 +22,62 @@
           'rabbit-exchange': os.getenv('LCW_RABBIT_EXCHANGE', 'local.lcmap.changes.worker'),
           'rabbit-result-routing-key': os.getenv('LCW_RABBIT_RESULT_ROUTING_KEY', 'change-detection-result'),
           'rabbit-ssl': os.getenv('LCW_RABBIT_SSL', False),
-          'ubid_band_dict' : {
+          'api-host': os.getenv('LCW_API_HOST', 'http://localhost:5678'),
```

Comment:
this isn't a host, it's a host and a port.  Recommend either splitting into either host, port, tile_spec_context or just replacing with tile_specs_url defaulted to http://localhost:5678/tile-specs